app/helpers/application_helper/button/ab_group_edit.rb

```diff
@@ -0,0 +1,25 @@
+# This class describes the behavior of both ab_group_edit and ab_group_delete
+# buttons, hence they differ only in the disable message.
```

Comment:
@martinpovolny I don't think it's a RBAC feature in this case, but yes, `GenricFeatureButton` is good example to parametrize a button